todoapp/TODOapp/app/src/main/java/com/nearsoft/androidschool/todoapp/activities/detail/DetailActivity.java

```diff
@@ -136,7 +184,7 @@ private void populateToDoObject() {
             todoItem = getTodo();
             displayDetail();
             buttonViewConfig(false);
-        } catch (AssertionError err) {
+        } catch (Exception e) {
```

Comment:
@jmsalcido the getTodo method used to thrown an Exception because in the beginning of the creation of this activity there was only one way to get to this activity, through selecting a ToDo from the list, meaning that the intent would contain extras and thrown an exception if not. Later we added the functionality to create one, so if there was no extra that would mean that the user was creating a new ToDo, so the exception was pointless